gossip/src/crds.rs

```diff
@@ -56,6 +57,26 @@ use {
 const CRDS_SHARDS_BITS: u32 = 12;
 // Number of vote slots to track in an lru-cache for metrics.
 const VOTE_SLOTS_METRICS_CAP: usize = 100;
+// Number of ending zero bits for crds signature to get reported to influx
+// mean new push messages received per minute per node
+//      testnet: ~500k,
+//      mainnet: ~280k
+// target: 1-2 signatures reported per minute
+// log2(250k) = ~17.9.
+const SIGNATURE_SAMPLE_TRAILING_ZEROS: u8 = 18;
+
+pub fn trailing_n_crds_signature_bits_are_zero(signature_bytes: &[u8], n: u8) -> bool {
```

Comment:
This shouldn't be `pub`.
Instead of `signature_bytes: &[u8]` I think it is better to pass a `Signature` and do the `.as_ref()` inside the function.
`n: u8` also seems redundant since it is always the constant above.

I think the function better be named something like `should_report_message` or something like that. The fact that it samples based on trailing zeros is just an implementation detail.